ethcore/types/src/transaction/transaction.rs

```diff
@@ -462,6 +462,24 @@ impl SignedTransaction {
 		self.sender
 	}
 
+	/// Returns transaction receiver, if any
+	pub fn receiver(&self) -> Option<Address> {
+		match self.transaction.unsigned.action {
+			Action::Create => None,
+			Action::Call(receiver) => Some(receiver),
+		}
+	}
+
+	// Reference to the plain text transaction
+	pub fn as_unsigned(&self) -> &Transaction {
```

Comment:
Since there are `Deref` implementations that method is not needed, you can directly access stuff from `Transaction` anyway.